powerlevel9k.zsh-theme

```diff
@@ -1312,8 +1318,11 @@ NEWLINE='
 }
 
 prompt_powerlevel9k_setup() {
+  # I decided to use the value below for better supporting 32-bit CPUs, since the previous value "99999999999" was causing issues on my Android phone, which is powered by an armv7l
+  # We don't have to change that until 19 January of 2038! :)
+
   # Disable false display of command execution time
-  _P9K_TIMER_START=99999999999
+  _P9K_TIMER_START=2147483647
```

Comment:
Could you add a comment, that this is the maximum integer on 32bit? <- Here as well